demo_path_routing_auth/urls.py

```diff
@@ -8,6 +8,9 @@
     url(r'^logout/$', views.logout),
     url(r'^launch/$', views.launch),
     url(r'^kill/(.*)$', views.kill),
+    url(r'^logs/(.*)$', views.logs),
+    url(r'^history/(.*)$', views.history),
+    url(r'^kill_lru/', views.kill_lru),
```

Comment:
is this a standard url  acronym

Reply:
Fairly standard, in some contexts: for example https://en.wikipedia.org/wiki/Cache_replacement_policies